pkg/heroku/heroku.go

```diff
@@ -250,14 +250,18 @@ type errorResp struct {
 	URL     string `json:"url"`
 }
 
-func checkResp(res *http.Response) error {
+func CheckResp(res *http.Response) error {
 	if res.StatusCode/100 != 2 { // 200, 201, 202, etc
 		var e errorResp
 		err := json.NewDecoder(res.Body).Decode(&e)
 		if err != nil {
 			return errors.New("Unexpected error: " + res.Status)
 		}
-		return Error{error: errors.New(e.Message), Id: e.Id, URL: e.URL}
+		ret := Error{error: errors.New(e.Message), Id: e.Id, URL: e.URL}
+		if e.Id == "message_required" {
+			panic(ret)
```

Comment:
Kinda sucks that we have to panic here. But I can't think of a much better way to do this check globally since the run actions don't return an error (although, that would probably be a good refactor later).